articles/governance/blueprints/create-blueprint-portal.md

```diff
@@ -114,7 +114,7 @@ Manager template and role assignment on the new resource group.
 
    ![Artifact - Resource Manager template](./media/create-blueprint-portal/add-resource-manager-template.png)
 
-1. Your completed blueprint should look similar to the following. Notice that each artifact has '_x_ out of _y_ parameters populated' under the _Parameters_ column. The **dynamic parameters** will be set during each assignment of the blueprint and the single **static parameter** on the role assignment has already been configured.
+1. Your completed blueprint should look similar to the following. Notice that each artifact has '_0_ out of _y_ parameters populated' under the _Parameters_ column. The **dynamic parameters** will be set during each assignment of the blueprint.
```

Comment:
While each example is _0_, **x of y** is the phrase when showing numbers in such a way.  Please change this back to an x.  The rest of the change on this line looks fine.